### PR TITLE
Fix coding style using autopep8

### PR DESCRIPTION
**Pull request by okuta**
I tried autopep8 `find . -name '*.py' -exec autopep8 --in-place '{}' \;` except caffe_pb2.py and caffe_pb3.py.

Seealso #1300.

### CODE REVIEW

**Comment by delta2323** [comment]
LGTM 👍

**Comment by delta2323** [comment]
Could you merge or rebase the master branch?

**Comment by unnonouno** [comment]
This fix causes a lot of conflicts with other PRs. Please wait till many PRs are merged.

**Comment by okuta** [comment]
OK, I understand.

**Comment by delta2323** [comment]
@unnonouno Could you suggest the appropriate day on which we merge this PR?

**Comment by unnonouno** [comment]
How about using `-r` option?
And, how about applying this command in traivis ci? We also need to fix contribution guide.

**Comment by okuta** [comment]
@delta2323 I fixed coding style.

**Comment by delta2323** [comment]
Thank you for fixing the code. It seems OK. So we should determine when to merge it.

**Comment by delta2323** [comment]
@okuta I and @unnonouno added checking mechanism by autopep8 in travis. As the PR originates from latest master branch, it introduced several codes that violates autopep8. Could you fix them?

**Comment by unnonouno** [comment]
OK, please go ahead.

**Comment by unnonouno** [comment]
Did you fix the contribution guide?

**Comment by delta2323** [comment]
> Did you fix the contribution guide?

He didn't seem to fix it.

**Comment by okuta** [comment]
@unnonouno I fixed. Please review this.

File: docs/source/contribution.rst [hunk 1]
```diff
@@ -94,15 +94,20 @@ Coding Guidelines
...
-  $ pip install hacking
+  $ pip install autopep8 hacking
+  $ autopep8 --global-config .pep8 path/to/your/code.py
   $ flake8 path/to/your/code.py
 
 To check Cython code, use ``.flake8.cython`` config file::
 
   $ flake8 --config=.flake8.cython path/to/your/cython/code.pyx
 
+The ``autopep8`` supports automatically format Python code to conform to the PEP 8 style guide::
```
Comment (delta2323) [inline docs/source/contribution.rst]: The autopep8 automatically correct Python code to conform to the ...

**Comment by delta2323** [comment]
LGTM 👍

**Comment by delta2323** [comment]
Ah, sorry. Please fix where I pointed.

**Pull request merged**